privacy-scaling-explorations/e2e-zk-ecdsa
# Add Demo script

# Description

Add end to end script to test:
1. Fetching an anonymity set from the query api (addresses that owns some min balance of an ERC20 token)
2. Create proof request
3. Submit proof request
4. Get proof response from proving server
5. Verify proof: TODO

Current script can be run in `./demo` with:
```
pnpm run prove -p [private key] -a [erc 20 address] -m [min balance integer]
```
